stage #1/module #8. Concurrency/lesson #1. Concurrency/content.md

```diff
@@ -0,0 +1,96 @@
+
+##  Concurrency
+
+
+
+Multithreading is a Java feature that allows concurrent execution of two or more parts of a program for maximum utilization of CPU. Each part of such program is called a thread. So, threads are light-weight processes within a process.
+
+**Commonly used methods of Thread class**
+
+1. **public void run()**: is used to perform action for a thread.
```

Comment:
I tend to think, that better to wrap part of code with `.

For example: `public void run()`